dashboard/src/app/colors/che-colors.styl

```diff
@@ -73,13 +73,20 @@ $che-button-primary-flat-disabled-shadow-color = alpha($stroke-color, 0.55)
 
 $che-delete-label-color = #EC8585
 
+// create-workspace flow
+$grey-input-label-color = #91979F
+$very-light-grey-background-color = #F7F7F7
+
```

Comment:
we defined color.json as application colors palette, now seems new are added, but need to deal - it's really new ones or replacement for existing